### ISSUE TITLE
Sizes of tensors must match except in dimension 2. Expected size 1 but got size 2 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by vivian686er** [Slickytail/ComfyUI-InstantX-IPAdapter-SD3#3]
Hi, brother, I encountered this error result after running：

“Sizes of tensors must match except in dimension 2. Expected size 1 but got size 2 for tensor number 1 in the list.”

Do you know what the reason is?

**Comment by vivian686er** [comment]
I copied this error to ChatGPT and it gave me this result：

“This error message indicates that a RuntimeError was encountered when executing ComfyUI, specifically a size mismatch problem occurred when processing tensor splicing. The error occurred in the forward method of the jointblock.py file, when trying to use torch.cat to splice two tensors (img_key and ip_key), the sizes of these two tensors must match except the second dimension (dim=2), but the program expects a dimension of the first tensor to be 1, but it is actually 2.”
